package.json

```diff
@@ -6,7 +6,6 @@
   "scripts": {
     "preinstall": "_scripts/check-package-manager.sh",
     "authors": "git shortlog -s | cut -c8- | sort -f > AUTHORS",
-    "audit": "echo 'audit is currently unavailable'",
```

Comment:
The first commit is removing an unrelated noop run script.

It looks like the script was a way to reduce `npm audit`’s frequency (https://github.com/mozilla/fxa/pull/4831), but the root package.json file’s `audit` command is a noop. Is there something that should be happening here that isn’t? If it’s not a clear cut answer, I will remove this first commit entirely.